datafusion-cli/Cargo.lock

```diff
@@ -2702,6 +2715,15 @@ dependencies = [
  "winapi",
 ]
 
+[[package]]
```

Comment:
for anyone following along, xz is now a new dependency

Reply:
maybe worth creating an optional feature?

Reply:
🤔  I wonder if we should make a feature for "support compression" as a similar argument could be made for bzip, etc

Reply:
feature gate added